test/test_listen_handler.py

```diff
@@ -6,35 +6,13 @@
 # Copyright (C) 2022 Jeny Sadadia
 # Author: Jeny Sadadia <jeny.sadadia@gmail.com>
 
-import pytest
-from api.main import app
-from api.models import User
-from fastapi.testclient import TestClient
-from unittest.mock import AsyncMock
-
-
-@pytest.fixture()
-def mock_get_current_user(mocker):
-    async_mock = AsyncMock()
-    mocker.patch('api.auth.Authentication.get_current_user',
-                 side_effect=async_mock)
-    return async_mock
+# pylint: disable=unused-argument
 
+"""Unit test functions for KernelCI API listen handler"""
 
-@pytest.fixture()
-def mock_init_sub_id(mocker):
-    async_mock = AsyncMock()
-    mocker.patch('api.pubsub.PubSub._init_sub_id',
-                 side_effect=async_mock)
-    return async_mock
-
-
-@pytest.fixture()
-def mock_listen(mocker):
-    async_mock = AsyncMock()
-    mocker.patch('api.pubsub.PubSub.listen',
-                 side_effect=async_mock)
-    return async_mock
+from fastapi.testclient import TestClient
```

Comment:
Maybe we can use this PR to sort imports alphabetically

Reply:
If I change import order to this,
```
from api.main import app
from api.models import User
from fastapi.testclient import TestClient
```

pylint will complain about wrong import order.
```
************* Module test.test_listen_handler
test/test_listen_handler.py:16:0: C0411: third party import "from fastapi.testclient import TestClient" should be placed before "from api.main import app" (wrong-import-order)

-------------------------------------------------------------------
Your code has been rated at 9.55/10 (previous run: 10.00/10, -0.45)
```